se_feedback/sprint_6/feedback_sprint_6_all.md

```diff
@@ -61,3 +61,23 @@ Total: ****
 | Code Review            |     |
 
 ##Notes
+* ADD
+	* The diagrams makes your explanation a lot clearer.
+	* You should explain how you run the tools in the Concurrency section 
```

Comment:
With the tools, do you mean how we run the ASATs? If so, we currently don't execute Maven on the user's project. We do explicitly say on the GUI that the user run `mvn site` before selecting his/her project.